open-metadata-distribution/open-metadata-assemblies/src/main/assemblies/omag-server.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+
+<assembly xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2"
+          xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2 http://maven.apache.org/xsd/assembly-1.1.2.xsd">
+    <formats>
+        <format>tar.gz</format>
+        <format>dir</format>
+    </formats>
+    <id>omag-server</id>
+    <baseDirectory>omag-server-${open-metadata.version}</baseDirectory>
+    <fileSets>
+        <fileSet>
+            <directory>../omag-server/target</directory>
+            <outputDirectory>.</outputDirectory>
+            <includes>
+                <include>*.jar</include>
+                <include>*.sh</include>
```

Comment:
We don't have any shell scripts currently. harmless, but could be unintended so I would remove until we need it?